node/sc/bridge_manager_test.go

```diff
@@ -152,6 +154,10 @@ func TestBridgeManager(t *testing.T) {
 	assert.NoError(t, err)
 	sim.Commit() // block
 
+	// Register tokens on the bridgeInfo
+	bridgeInfo.RegisterToken(tokenAddr, tokenAddr)
```

Comment:
The name of the BridgeInfo's `RegisterToken()` is confusing. How about adding `bridge.RegisterToken()` inside the method in future PR?

Reply:
It's good. I will refactor the below code as you commented.
```
func (sbapi *SubBridgeAPI) RegisterToken(cBridgeAddr, pBridgeAddr, cTokenAddr, pTokenAddr common.Address) error {
	if !sbapi.sc.bridgeManager.IsValidBridgePair(cBridgeAddr, pBridgeAddr) {
		return ErrInvalidBridgePair
	}

	cBi, cExist := sbapi.sc.bridgeManager.GetBridgeInfo(cBridgeAddr)
	pBi, pExist := sbapi.sc.bridgeManager.GetBridgeInfo(pBridgeAddr)

	if !cExist || !pExist {
		return errors.New("bridge does not exist")
	}

	err := cBi.RegisterToken(cTokenAddr, pTokenAddr)
	if err != nil {
		return err
	}

	err = pBi.RegisterToken(pTokenAddr, cTokenAddr)
	if err != nil {
		return err
	}

	cBi.account.Lock()
	tx, err := cBi.bridge.RegisterToken(cBi.account.GetTransactOpts(), cTokenAddr, pTokenAddr)
	if err != nil {
		cBi.account.UnLock()
		return err
	}
	cBi.account.IncNonce()
	cBi.account.UnLock()
	logger.Debug("scBridge registered token", "txHash", tx.Hash().String(), "scToken", cTokenAddr.String(), "mcToken", pTokenAddr.String())

	pBi.account.Lock()
	tx, err = pBi.bridge.RegisterToken(pBi.account.GetTransactOpts(), pTokenAddr, cTokenAddr)
	if err != nil {
		pBi.account.UnLock()
		return err
	}
	pBi.account.IncNonce()
	pBi.account.UnLock()
	logger.Debug("mcBridge registered token", "txHash", tx.Hash().String(), "scToken", cTokenAddr.String(), "mcToken", pTokenAddr.String())

	logger.Info("Register token", "scToken", cTokenAddr.String(), "mcToken", pTokenAddr.String())
	return nil
}
```